apisix/schema_def.lua

```diff
@@ -555,6 +580,14 @@ _M.ssl = {
             type = "integer",
             minimum = 1588262400,  -- 2020/5/1 0:0:0
         },
+        labels = {
```

Comment:
I think ssl may not need labels

Reply:
@nic-chen  IMHO, we could use label to distinguish among dozens of SSLs according to their usage.

Reply:
@membphis  @liuxiran  any thoughts on this? 😄

Reply:
> we could use label to distinguish among dozens of SSLs according to their usage.

Agree this. Searching by category will be helpful.

Reply:
> we could use label to distinguish among dozens of SSLs according to their usage

agree +1

Reply:
OK, I think it could work with SNI.But for management, labels is helpful for it.